core/DataArray.php

```diff
@@ -80,7 +80,7 @@ public static function makeEmptyRow()
      *
      * @return void
      */
-    protected function doSumVisitsMetrics($newRowToAdd, &$oldRowToUpdate, $onlyMetricsAvailableInActionsTable = false)
+    protected function doSumVisitsMetrics($newRowToAdd, &$oldRowToUpdate)
```

Comment:
Instead of a boolean parameter splitted logic into `doSumVisitsMetrics ` and `doSumActionMetrics`